sdk/communication/azure-communication-callautomation/azure/communication/callautomation/_models.py

```diff
@@ -204,6 +207,98 @@ def _to_generated(self):
                 play_source_id=self.play_source_id
             )
 
+class TextSource(object):
```

Comment:
You have added new models but it is not exposed in init. This should be shown in init so the developers dont have to access these class in _models